include/ck/tensor_operation/gpu/element/element_wise_operation.hpp

```diff
@@ -375,53 +375,58 @@ struct ScaleAddScaleAddRelu
     }
 
     template <typename E, typename C, typename D0, typename D1>
-    __host__ __device__ constexpr void
-    operator()(E& e, const C& c, const D0& d0, const D1& d1) const;
+    __host__ __device__ void operator()(E& e, const C& c, const D0& d0, const D1& d1) const;
```

Comment:
Does removing constexpr here affect static elementwise ops?

Reply:
Generally I think constexpr keyword here is nonsense because you can't perform Relu operations etc. during compilation (the exception would be operations for Number<> but we don't have such reloads anyway), I'll try to add some measurements.

Reply:
Ok, I restored constexpr keyword to be 100% safe. I also measured scaleaddscaleadd from convnd_active example and perforamnce looks the same like before change. Thanks for catch.